### PR TITLE
Adding support for `kitty` terminal

### PR DESCRIPTION
**Pull request by spacekookie**
`kitty` is a modern terminal similar to `alacritty`, except with a cuter name and proper ligature support :grin:

### CODE REVIEW

File: rc/base/x11.kak [hunk 1]
```diff
@@ -4,6 +4,7 @@ declare-option -docstring %{shell command run to spawn a new terminal
 A shell command is appended to the one set in this option at runtime} \
     str termcmd %sh{
     for termcmd in 'alacritty      -e sh -c' \
+                   'kitty          -e sh -c' \
```
Comment (SolitudeSF) [inline rc/base/x11.kak]: `-e` is redundant for kitty

**Comment by spacekookie** [comment]
Now the CI fails, although it looks like an unrelated issue... 🤔

**Comment by lenormf** [comment]
I think what you want for command is `kitty /bin/sh -c`.

**Comment by SolitudeSF** [comment]
other entries dont use the full path tho

**Comment by spacekookie** [comment]
It appears `kitty` just ignores the `-e` option, because the original change is what's currently running on my system. Nonetheless, I hope this passes CI now :slightly_smiling_face:

**Comment by mawww** [comment]
Hello,

could you squash all those commits into a single one please ?

Cheers,

Maxime.

**Comment by spacekookie** [comment]
@mawww Is that directed towards me or the reviewer? :grin: If it's the former: Can't_ you just squash merge via the github UI?

**Comment by mawww** [comment]
@spacekookie I believe only you (the author) have the ability to modify the pull request, I am not sure if the github UI provides a way to do that.

**Comment by lenormf** [comment]
`$ git rebase -i 38b0bae~1` then `j<a-]>p <a-s>gi ess<esc>`.

**Comment by spacekookie** [comment]
Right, this should be ok now? Whyever it closed the PR before :thinking:

**Pull request merged**